src/ast.c

```diff
@@ -575,7 +575,7 @@ static jl_value_t *scm_to_julia_(fl_context_t *fl_ctx, value_t e, jl_module_t *m
             assert(jl_is_symbol(ex));
             temp = jl_module_globalref(jl_core_module, (jl_sym_t*)ex);
         }
-        else if (sym == inert_sym || (sym == quote_sym && (!iscons(car_(e))))) {
+        else if ((sym == inert_sym || (sym == quote_sym && (!iscons(car_(e))))) && iscons(e)) {
```

Comment:
`iscons(e)` needs to be checked before `car_(e)`; `car` is only valid if `e` is a cons.